man/man1/ucx_info.1

```diff
@@ -0,0 +1,93 @@
+.\"
+.\" Copyright (C) Mellanox Technologies Ltd. 2001-2011.  ALL RIGHTS RESERVED.
```

Comment:
As for now I think only SBU copyright should be here

Reply:
Was anticipating future (soon) changes...